init/arch_specs/eessi_arch_arm.spec

```diff
@@ -1,5 +1,6 @@
 # ARM CPU architecture specifications (see https://gpages.juszkiewicz.com.pl/arm-socs-table/arm-socs.html for guidance)
 # Software path in EESSI 	| Vendor ID 	| List of defining CPU features
+"aarch64/a64fx"	""		"asimdhp sve"		# Fujitsu A64FX
```

Comment:
Hmm, this doesn't fully work, because `asimdhp` is also available for `neoverse_v1`, which also has features that start with `sve` (like `svei8mm`).

The CPU features check should be checking for *exact* strings though, and not consider `svei8mm` as a valid match for `sve`...

Reply:
I think both have `sve` according to archspec:
```bash
[malaca@cna0002 A64FX_auto_detection]$ python3 -c "from archspec import cpu;import archspec; print(archspec.cpu.TARGETS['neoverse_v1'].features)"
{'flagm', 'pmull', 'rng', 'asimd', 'dgh', 'bf16', 'cpuid', 'asimdfhm', 'evtstrm', 'fphp', 'jscvt', 'ilrcpc', 'fp', 'i8mm', 'asimdrdm', 'dcpop', 'dcpodp', 'aes', 'sha2', 'asimdhp', 'fcma', 'sha3', 'crc32', 'svebf16', 'sha1', 'uscat', 'sha512', 'lrcpc', **'sve'**, 'dit', 'asimddp', 'svei8mm', 'atomics'}
```

Reply:
The problem is that we'll need to take into account the processor vendor to discriminate them, just looking at CPU features is not enough here...

Reply:
From what I tested locally, adding 0x46 in the Vendor ID solves both a64fx and graviton 3 identifications. You still need to create a file: tests/archdetect/aarch64/a64fx/Deucalion-Rocky85.output thate should read `aarch64/a64fx:aarch64/generic`